IFC4x3/Properties/p/ProductionType_0YHtL_$tbF_x44ptpVtMSn/DocProperty.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="ProductionType_0YHtL_tbF_x44ptpVtMSn" Name="ProductionType" UniqueId="2247757e-ff79-4ffb-b104-cf7cdfdd6731" PropertyType="p_enumeratedvalue" PrimaryDataType="IfcLabel">
```

Comment:
Does this have to be an enum? I would propose reusing `ConstructionMethod`:IfcLabel

Reply:
Ok Commit :

Reply:
Ok commit : de4bcf9de7f5e2e1d9e24e66e3b338da3b5506eb